build_info/pacman.control

```diff
@@ -3,7 +3,7 @@ Version: @DEB_PACMAN_V@
 Architecture: @DEB_ARCH@
 Maintainer: @DEB_MAINTAINER@
 Section: Utilities
-Depends: libarchive13, libarchive-tools, openssl, libcurl4, libzstd1, tar, libintl8
+Depends: libarchive13, libarchive-tools, openssl, libcurl4, libzstd1, tar, libintl8, bash-completion
 Priority: optional
 Description: A library-based package manager with dependency support
 Name: makepkg
```

Comment:
Why is this here to begin with?

Reply:
This is because Hayden didn’t want people to install it and mess up their device, labeling it as `makepkg` well make less people recognize it as the package manager thinking they can use it in parity with `dpkg`, but instead as `makepkg`, which is the only reason to install this.

Reply:
https://github.com/ProcursusTeam/Procursus/pull/102#issuecomment-673544226

Reply:
This could have been made clear in the description. The name of the package is pacman, and considering that we often follow the naming based on what Debian/Arch uses, the name flag isn't needed.